pygef/broxml.py

```diff
@@ -24,7 +23,7 @@ def __init__(self, path: str = None, string: str = None):
         ), "One of [path, string] should be not none."
         self.path = path
         if string is None and path is not None:
-            with open(path, encoding="utf-8", errors="ignore") as f:
+            with open(path, "rb") as f:
```

Comment:
Shouldn't the `encoding="utf-8"` be specified here?

Reply:
`etree.fromstring` doesn't work with utf-8, only binary, but maybe indeed I should create a different attribute for it, otherwise it is confusing.